data/campaigns/Eastern_Invasion/scenarios/16_Eleventh_Hour.cfg

```diff
@@ -0,0 +1,2237 @@
+    {TOWNSFOLK 21 20 dandy.png   18 20}
+
+    {TOWNSFOLK 26 28 nobleF.png  34 25} # south
+    {TOWNSFOLK 33 30 worker.png  22 31}
+    {TOWNSFOLK 29 31 workerF.png 22 31}
+
+    {TOWNSFOLK 37 19 madame.png  34 18} # northeast
+    {TOWNSFOLK 39 23 noble.png   35 26}
+    {TOWNSFOLK 42 14 crier.png   43 20}
+
+    #--------------------
+    # FLAVOR
+    #--------------------
+    [event]
+        name=moveto
+
+        [filter]
+            id=Addogin
+        [/filter]
+
+        [message]
+            speaker=Addogin
+            message= _ "So this is Weldyn? Didn’t know cities got this big. Never thought I’d end up fightin’ here, of all places."
+        [/message]
+    [/event]
+    [event]
+        name=moveto
+
+        [filter]
+            id=Gaennell
+        [/filter]
+
+        [message]
+            speaker=Mel Guthrak
+            message= _ "Gaennell, you traitor! <b>I</b> am your master, not some petty king of men. Return to me and face your punishment."
+        [/message]
+        [message]
+            speaker=Gaennell
+            message= _ "You tried to have me killed over a petty accident! I’m not going to grovel at your feet and suffer your abuse any more."
+        [/message]
+    [/event]
+    [event]
+        name=attack
+
+        [filter]
+            id=Yannic
+        [/filter]
+
+        [message]
+            speaker=Yannic
+            message= _ "For my fallen brothers at Soradoc and the Plains!"
+        [/message]
+    [/event]
+    [event]
+        name=attack
+
+        [filter]
+            id=Konrad
+        [/filter]
+
+        [filter_second]
+            [not]
+                id=Mal-Ravanal
+            [/not]
+        [/filter_second]
+
+        [filter_condition]
+            {VARIABLE_CONDITIONAL konrad_spoke not_equals yes}
+        [/filter_condition]
+        # reused in S17b, if he doesn't speak here
+        [message]
+            speaker=Konrad
+            message= _ "My legacy will not be one of failure! With all the might of the throne of Wesnoth, I will strike you down!"
+        [/message]
+        {VARIABLE konrad_spoke yes}
+    [/event]
+
+    #--------------------
+    # CAPTURE A GENERAL'S VILLAGE
+    #--------------------
+    [event]
+        name=capture
+        [filter]
+            side=1
+        [/filter]
+        [filter_condition]
+            [variable]
+                name=owner_side
+                less_than_equal_to=4
+            [/variable]
+            [lua] # only at first, in case the player wants to take them all immediately
+                code = << return wml.variables.turn_number<=5 >>
+            [/lua]
+        [/filter_condition]
+
+        {FIND_NEARBY (side,canrecruit=$owner_side,yes) 30 22 99}
+        [message]
+            speaker=$found_unit.id
+            message= _ "My King, you are of course welcome to capture any of my villages if you need them, but please know that it will make it more difficult to heal my injured soldiers."
```

Comment:
I understand this is explanatory dialogue framed in the context of the story but perhaps 'welcome to take any of my villages' makes more sense. I imagine a king (assuming the player is playing on behalf of Konrad) does not 'capture' what is already his own.

Reply:
Addressing the king as "My King" seems disrespectful. Should it not be "Your Majesty" or "Sire" instead? Maybe "My liege" if you know for sure that the speaker is a citizen of the Kingdom of Wesnoth.